elements/pfe-badge/BaseBadge.ts

```diff
@@ -0,0 +1,41 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { pfelement } from '@patternfly/pfe-core/decorators.js';
+
+import style from './BaseBadge.scss';
+
+export abstract class BaseBadge extends LitElement {
+  static readonly styles = [style];
+
+  /**
+   * Denotes the state-of-affairs this badge represents
+   * Options include read and unread
+   */
+  @property({ reflect: true }) state?: 'default'|'unread'|'read' = 'default';
```

Comment:
```suggestion
  @property({ reflect: true }) state?: 'unread'|'read';
```
Let's avoid initializing reflected properties. We also don't really need the default state, we can just assume that as the default